Orange/widgets/data/owselectcolumns.py

```diff
@@ -316,6 +359,46 @@ def update_domain_role_hints(self):
         hints.update(hints_from_model("meta", self.meta_attrs))
         self.domain_role_hints = hints
 
+    @Inputs.features
+    def set_features(self, features):
+        self.features = features
+
+    def handleNewSignals(self):
+        self.check_data()
+        self.enable_used_attrs()
+        self.enable_use_features_box()
+        if self.use_input_features and len(self.features_from_data_attributes):
```

Comment:
Since `self.features_from_data_attributes` is a list, PEP8 recommends `if self.use_input_features and self.features_from_data_attributes`